<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddGardenTool</class>
 <widget class="QDialog" name="AddGardenTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>The Rose Garden Nursery  System 1.0</string>
  </property>
  <property name="styleSheet">
   <string notr="true"> background-color: rgb(56, 144, 96);
background-image: url(:/images/beauty.png);</string>
  </property>
  <widget class="QPushButton" name="btn_View_Record">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{
color: rgb(255, 255, 255);

}</string>
   </property>
   <property name="text">
    <string>View Record</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Insert_Record">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>40</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Insert Data</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Delete_Record">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Delete Record</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Update_Record">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string> Update Record</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>101</y>
     <width>731</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">  QTableWidget 
{

  border:3px solid gray;
  border-radius: 10px;
  background: rgb(255, 255, 255);
 
  color: rgb(0, 0, 0);;

}</string>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Item Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Quantity</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Status</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Description</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ID">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>90</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
 QLineEdit {
  border:3px solid gray;
  border-radius: 10px;
  padding: 8px 8px;
  background: rgb(255, 255, 255);
  font: 500 12pt ;
  color: rgb(0, 0, 0);;
}</string>
   </property>
   <property name="placeholderText">
    <string>Enter ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Item_name">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>140</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
 QLineEdit {
  border:3px solid gray;
  border-radius: 10px;
  padding: 8px 8px;
  background: rgb(255, 255, 255);
  font: 500 12pt ;
  color: rgb(0, 0, 0);;
}</string>
   </property>
   <property name="placeholderText">
    <string>Item Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Quantity">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>190</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
 QLineEdit {
  border:3px solid gray;
  border-radius: 10px;
  padding: 8px 8px;
  background: rgb(255, 255, 255);
  font: 500 12pt ;
  color: rgb(0, 0, 0);;
}</string>
   </property>
   <property name="placeholderText">
    <string>Quantity</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Status">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>240</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
 QLineEdit {
  border:3px solid gray;
  border-radius: 10px;
  padding: 8px 8px;
  background: rgb(255, 255, 255);
  font: 500 12pt ;
  color: rgb(0, 0, 0);;
}</string>
   </property>
   <property name="placeholderText">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Description">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>290</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
 QLineEdit {
  border:3px solid gray;
  border-radius: 10px;
  padding: 8px 8px;
  background: rgb(255, 255, 255);
  font: 600 12pt ;
  color: rgb(0, 0, 0);;
}</string>
   </property>
   <property name="placeholderText">
    <string>Description</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
